packages/pangea-node-sdk/src/services/authn/user/authenticators.ts

```diff
@@ -24,9 +24,7 @@ export default class UserAuthenticators extends BaseService {
    * ```
    */
   delete(
-    request:
-      | AuthN.User.Authenticators.Delete.EmailRequest
-      | AuthN.User.Authenticators.Delete.IDRequest
+    request: AuthN.User.Authenticators.Delete.Request
```

Comment:
Ins't this a breaking change? Does `AuthN.User.Authenticators.Delete.Request` include the older classes?

Reply:
It's equivalent as far as this parameter is concerned, I've just consolidated what would have been 3 interfaces into 1 interface + 1 type.

Reply:
Got it, but my question is... if someone has something like 
```
request: AuthN.User.Authenticators.Delete.EmailRequest {
   email: "foo@bar",
}
```
Will this still work or it will complain because it now takes a 
`request: AuthN.User.Authenticators.Delete.Request`

Reply:
I don't think we document directly referencing the interfaces like that but since they are indeed public, sure, I have restored them all.